app/models/admin_menu.rb

```diff
@@ -26,7 +26,7 @@ class AdminMenu
     scope :customization, "tools-line", [
       item(name: "config"),
       item(name: "html variants", controller: "html_variants"),
-      item(name: "display ads"),
+      item(name: "billboards"),
```

Comment:
For my part, I would say, if we're going to change _anything_ the user sees, we probably should change _everything_ the user sees on the same page together. Or, rather: _if_ we want this PR to be small & incremental, maybe we leave the link label "display ads" (thus changing only class names & routes, all behind-the-scenes) ?

<img width="1103" alt="Screenshot 2023-07-06 at 08 49 27" src="https://github.com/forem/forem/assets/2077/e76b4e44-6f75-4e02-b3b5-0457fd63a18d">

Reply:
In general I agree, I wasn't planning to change the lable at this point, but had to because it is related to the new routes (didn't want to complicate things/add an option or smth just for transition).
I will evaluate how large is changing "anything the user sees" on this page.

Reply:
Ok, I decided to go with "incremental changes" and just added a controller option :)